### DISCUSSION TITLE
Shortcut to open multiple PDFs in a specific folder

### DISCUSSION THREAD

**Discussion by Keben73** [sumatrapdfreader/sumatrapdf#3630] [Q&A]
Since tab groups are not allowed, and I can't seem to find a workaround to making tab groups happen. I was wondering how to open a shortcut to open all PDF in a specific folder. I stumbled upon [-appdata ""C:\Path\to\Your\PDF"], but it doesn't seem to work. I also looked at some similar discussions, but still can't get it to work. It only shows the main homescreen.

How can I fix this? Or how do you guys manage not having tab groups?

**Comment by GitHubRulesOK** [comment]
-appdata points to separate folder with separate settings file
 so there can be a settings file for all comics folder or settings in each folder group
 the usage is per user for user controlled sets (no transfer of history between settings files.)

To open a folder of IMAGES you can drag and drop, but it is not intended that folders are generally "opened" and some filetypes can cause blocking actions to traversing from folder file to folder file.
